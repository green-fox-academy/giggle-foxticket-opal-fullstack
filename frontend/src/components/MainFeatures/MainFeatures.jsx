import React from 'react';
import './MainFeatures.styles.scss';

const MainFeatures = () => {
  return (
    <div className="main-features">
      <div className="card">
        <div className="card-image">
          <img
            src="https://cdn.vox-cdn.com/thumbor/m7yu15u1ImfzuROkTkv3OKt68fY=/1400x1050/filters:format(png)/cdn.vox-cdn.com/assets/3280553/doom-560.png"
            alt=""
          />
        </div>
        <div className="card-copy">
          <h1>DOOM</h1>
          <h2>Video game series</h2>
          <p>
            Doom is a video game series and media franchise created by John
            Carmack, John Romero, Adrian Carmack, Kevin Cloud, and Tom Hall. The
            series focuses on the exploits of an unnamed space marine operating
            under the auspices of the Union Aerospace Corporation, who fights
            hordes of demons and the undead.
          </p>
        </div>
      </div>

      <div className="card">
        <div className="card-image">
          <img
            src="https://cdn.vox-cdn.com/thumbor/m7yu15u1ImfzuROkTkv3OKt68fY=/1400x1050/filters:format(png)/cdn.vox-cdn.com/assets/3280553/doom-560.png"
            alt=""
          />
        </div>
        <div className="card-copy">
          <h1>DOOM</h1>
          <h2>Video game series</h2>
          <p>
            Doom is a video game series and media franchise created by John
            Carmack, John Romero, Adrian Carmack, Kevin Cloud, and Tom Hall. The
            series focuses on the exploits of an unnamed space marine operating
            under the auspices of the Union Aerospace Corporation, who fights
            hordes of demons and the undead.
          </p>
        </div>
      </div>

      <div className="card">
        <div className="card-image">
          <img
            src="https://cdn.vox-cdn.com/thumbor/m7yu15u1ImfzuROkTkv3OKt68fY=/1400x1050/filters:format(png)/cdn.vox-cdn.com/assets/3280553/doom-560.png"
            alt=""
          />
        </div>
        <div className="card-copy">
          <h1>DOOM</h1>
          <h2>Video game series</h2>
          <p>
            Doom is a video game series and media franchise created by John
            Carmack, John Romero, Adrian Carmack, Kevin Cloud, and Tom Hall. The
            series focuses on the exploits of an unnamed space marine operating
            under the auspices of the Union Aerospace Corporation, who fights
            hordes of demons and the undead.
          </p>
        </div>
      </div>
    </div>
  );
};

export default MainFeatures;
