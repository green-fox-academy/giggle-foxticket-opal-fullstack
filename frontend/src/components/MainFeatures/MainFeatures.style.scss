.main-features {
  display: flex;
  justify-content: space-between;
  margin: 5% 15% 5% 15%;
  overflow: hidden;

  div {
    display: flex;
    justify-content: center;
    // align-items: center;
    background: gainsboro;
    border-radius: 5px;
    border: 1px solid rgb(197, 190, 190);
    width: 300px;
    height: 320px;
    padding: 5px;
    position: relative;

    h3 {
      position: absolute;
    }

    p {
      margin-top: 20%;
    }
  }
}
