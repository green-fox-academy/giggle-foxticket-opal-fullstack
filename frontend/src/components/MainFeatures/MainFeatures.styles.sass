.main-features
  display: flex
  justify-content: space-between
  margin: 5%

h1,
h2
  margin: 0
  text-align: left


h2
  font-style: italic
  font-size: 15px
  color: darkgray


.card
  background: whitesmoke
  margin: 50px auto
  max-width: 400px
  text-align: center
  border-radius: 10px
  box-shadow: 10px 10px 30px #bdbdbd


.card-image
  border-radius: 10px 10px 0 0
  overflow: hidden
  max-height: 200px

  img
    width: 400px


.card-copy
  padding: 20px
