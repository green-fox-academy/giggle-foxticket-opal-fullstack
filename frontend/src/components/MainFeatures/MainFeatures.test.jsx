import React from 'react';
import { shallow } from 'enzyme';
import MainFeatures from './MainFeatures';

it('expect to render Main Features component', () => {
  expect(shallow(<MainFeatures />)).toMatchSnapshot();
});
