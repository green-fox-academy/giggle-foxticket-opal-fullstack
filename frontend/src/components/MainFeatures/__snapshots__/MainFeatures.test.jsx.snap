// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Main Features component 1`] = `ShallowWrapper {}`;
